docs/ske/10-integrations/20-port.mdx

```diff
@@ -0,0 +1,347 @@
+By installing this via the SKE Controller you will get configuration and management of
+Kratix Promises and Resources automatically managed for you, including managing the
+backend of your choosing for the self-service actions in Port.
+
+
+### Prerequisites 
+
+To install the SKE Port Controller, you will need:
+
+* A SKE token. To request one, visit [Syntasso](https://www.syntasso.io/pricing) and click "Try SKE
+Today".
+* A Kubernetes cluster with Kratix running. The [Kratix Quick Start](../../main/quick-start)
+is an easy way to get started.
+
+
+### Configure the integration
+
+To configure the controller, you will need to create a configmap and a secret in the
+`ske-port-controller-system` namespace.
+
+Start by creating this namespace if you have not previously installed the controller.
+
+Then create local environment variables for the three required values:
+* `PORT_CLIENT_ID`
+* `PORT_CLIENT_SECRET`
+* `PORT_GITHUB_ORG`
+
+And review the remaining default configurations. If you require overriding any, set those
+local environment variables as well.
+
+* `PORT_GITHUB_REPO` (default: port-syncer)
+* `PORT_GITHUB_WORKFLOW_FILE` (default: port-action.yml)
+* `PORT_CM_NAME` (default: ske-port-k8s-exporter)
+* `PORT_CM_NAMESPACE` (default: ske-port-controller-system)
+* `PORT_SKIP_DELETE_BLUEPRINT` (default: false)
+
+Once environment variables have been set, you can create the configmap using the
+following command:
+
+```bash
+kubectl create secret generic -n ske-port-controller-system ske-port-exporter-config \
+    --from-literal="PORT_CLIENT_ID=${PORT_CLIENT_ID}" \
+    --from-literal="PORT_CLIENT_SECRET=${PORT_CLIENT_SECRET}" \
+    --from-literal="PORT_GITHUB_ORG=${PORT_GITHUB_ORG}"
+    --from-literal="PORT_GITHUB_REPO=${PORT_GITHUB_REPO:-}" \
+    --from-literal="PORT_GITHUB_WORKFLOW_FILE=${PORT_GITHUB_WORKFLOW_FILE:-}" \
+    --from-literal="PORT_CM_NAME=${PORT_CM_NAME:-}" \
+    --from-literal="PORT_CM_NAMESPACE=${PORT_CM_NAMESPACE:-}" \
+    --from-literal="PORT_SKIP_DELETE_BLUEPRINT=${PORT_SKIP_DELETE_BLUEPRINT:-}"
+```
+
+### Access the enterprise controller image
+
+Next you can either pre-load the required image into your cluster, or define your
+authentication credentials via a docker-registry secret and refer to this secret in
+Pipeline definition in your Workflow.
+
+The image can be pulled after replacing VERSION with the target version from the
+[releases page](../releases/ske-port-controller):
+
+```bash
+docker pull ghcr.io/syntasso/ske-port-controller:<VERSION>
+```
+
+If you prefer an to use `imagePullSecret`, you can create the secret with the following
+command, replacing YOUR_TOKEN with the token you have been provided with:
+
+```
+kubectl create secret docker-registry syntasso-registry \
+  --docker-server=registry.syntasso.io \
+  --docker-username=syntasso-pkg \
+  --docker-password=YOUR_TOKEN \
+  --namespace kratix-platform-system
+```
```

Comment:
should we just document using the image pull secret, but then have a note/info saying if your using a cluster where you can manually pre-load the iomage, e.g. kind, then just docker pull and do that?

Reply:
also the image pull secret ref is hardcoded in the manifest, not sure how this works if your pre-loading and not having the secret present?